vncdotool/client.py

```diff
@@ -344,7 +344,14 @@ def mouseDrag(self, x, y, step=1):
     def setImageMode(self):
         """ Extracts color ordering and 24 vs. 32 bpp info out of the pixel format information
         """
-        if (self.truecolor and (not self.bigendian) and self.depth == 24
+        if self._version_server == 3.889:
```

Comment:
It would be useful to have a comment why this quirk exists, rather than requiring a reader to view the history, it could be as simple as linking back to this PR.